src/components/icon/utils.ts

```diff
@@ -140,17 +140,3 @@ export const inheritAttributes = (el: HTMLElement, attributes: string[] = []) =>
 
   return attributeObject;
 }
-
-/**
- * Returns `true` if the document or host element
- * has a `dir` set to `rtl`. The host value will always
- * take priority over the root document value.
- */
-export const isRTL = (hostEl?: Pick<HTMLElement, 'dir'>) => {
```

Comment:
is this okay to remove? it's not used anywhere else

Reply:
Yeah we can remove this if we aren't using it anymore.